ttnn/ttnn/operations/normalization.py

```diff
@@ -22,6 +22,27 @@ def _golden_function(input_tensor: ttnn.Tensor, dim: int, **_):
     golden_function=_golden_function,
 )(ttnn._ttnn.operations.normalization.softmax)
 
+softmax_in_place = ttnn.register_operation(
+    golden_function=_golden_function,
+)(ttnn._ttnn.operations.normalization.softmax_in_place)
+
+scale_mask_softmax_in_place = ttnn.register_operation(
+    golden_function=_golden_function,
+)(ttnn._ttnn.operations.normalization.scale_mask_softmax_in_place)
+
+scale_mask_softmax = ttnn.register_operation(
+    golden_function=_golden_function,
```

Comment:
its golden is torch.softmax?

Reply:
same as above, should we change them to the same API now?